src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -140,7 +147,6 @@ public function configureOptions(OptionsResolver $resolver)
                         $options['em'],
                         $options['class'],
                         $qbParts,
-                        $options['loader'],
```

Comment:
it was not making caching useless, as ``$options['loader']`` was set only in case the option was used explicitly (not the case by default.

I agree that the loader returned by ``getLoader`` does not belong to the cache key: the other parts of the cache key are what determine this loader, and this is why they are here